pex/compatibility.py

```diff
@@ -29,8 +29,6 @@
 PY3 = sys_version_info[0] == 3
 
 string = (str,) if PY3 else (str, unicode)  # type: ignore[name-defined]
-unicode_string = (str,) if PY3 else (unicode,)  # type: ignore[name-defined]
-bytes = (bytes,)  # type: ignore[has-type]
```

Comment:
This shadowed the builtin `bytes`, which messed up below return types. It was also unused.